src/DETHRACE/common/utility.c

```diff
@@ -144,45 +142,52 @@ void EncodeLine(char* pS) {
         }
     }
     while (len > 0 && (pS[len - 1] == '\r' || pS[len - 1] == '\n')) {
-        --len;
-        pS[len] = 0;
+        len--;
+        pS[len] = '\0';
     }
 
     seed = len % 16;
+
     for (i = 0; i < len; i++) {
         c = pS[i];
+#if defined(DETHRACE_FIX_BUGS)
         if (i >= 2) {
             if (pS[i - 1] == '/' && pS[i - 2] == '/') {
                 key = (char*)gOther_long_key;
             }
         }
+#endif
         if (gEncryption_method == 1) {
             if (c == '\t') {
-                c = 0x80;
+                c = 0x9f;
```

Comment:
hmm, good spotting. Dont know how I got encryption methods round the wrong way! 🤔